tools/ncf.py

```diff
@@ -430,9 +477,10 @@ def generate_technique_content(technique_metadata):
       arg_value = ', '.join(args)
     else:
       arg_value = ""
-    
+    class_context = canonify_class_context(method_call['class_context'])
+
     content.append('    "method_call" usebundle => '+method_call['method_name']+'('+arg_value+'),')
-    content.append('      ifvarclass => "'+method_call['class_context']+'";')
+    content.append('      ifvarclass => concat("'+class_context+'");')
```

Comment:
quote there ? are you sure ? 
wouldn't it generate
concat(""Monday.",canonify(${bundle2.var}),".debian.",canonify(${bundle.var}),".linux"")